Description: Made changes using style
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 python3-huggingface-hub (0.29.3-1) UNRELEASED; urgency=medium
 .
   * Initial release. (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: anudeep <anudeep@unknown>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2025-03-24

--- python3-huggingface-hub-0.29.3.orig/Makefile
+++ python3-huggingface-hub-0.29.3/Makefile
@@ -9,8 +9,8 @@ quality:
 	ruff format --check $(check_dirs) # formatter
 	PYTHONPATH=$(CURDIR)/src python utils/check_inference_input_params.py
 	python utils/check_contrib_list.py
-	python utils/check_static_imports.py
-	python utils/check_all_variable.py
+	PYTHONPATH=$(CURDIR)/src python utils/check_static_imports.py
+	PYTHONPATH=$(CURDIR)/src python utils/check_all_variable.py
 	python utils/generate_async_inference_client.py
 
 	mypy src
@@ -19,8 +19,8 @@ style:
 	ruff format $(check_dirs) # formatter
 	ruff check --fix $(check_dirs) # linter
 	python utils/check_contrib_list.py --update
-	python utils/check_static_imports.py --update
-	python utils/check_all_variable.py --update
+	PYTHONPATH=$(CURDIR)/src python utils/check_static_imports.py --update
+	PYTHONPATH=$(CURDIR)/src python utils/check_all_variable.py --update
 	python utils/generate_async_inference_client.py --update
 
 inference_check:
--- python3-huggingface-hub-0.29.3.orig/src/huggingface_hub/inference/_generated/_async_client.py
+++ python3-huggingface-hub-0.29.3/src/huggingface_hub/inference/_generated/_async_client.py
@@ -327,10 +327,7 @@ class AsyncInferenceClient:
 
                 try:
                     response = await session.post(
-                        request_parameters.url,
-                        json=request_parameters.json,
-                        data=data_as_binary,
-                        proxy=self.proxies,
+                        request_parameters.url, json=request_parameters.json, data=data_as_binary, proxy=self.proxies
                     )
                     response_error_payload = None
                     if response.status != 200:
@@ -2149,12 +2146,7 @@ class AsyncInferenceClient:
         truncate: Optional[int] = None,
         typical_p: Optional[float] = None,
         watermark: Optional[bool] = None,
-    ) -> Union[
-        str,
-        TextGenerationOutput,
-        AsyncIterable[str],
-        AsyncIterable[TextGenerationStreamOutput],
-    ]:
+    ) -> Union[str, TextGenerationOutput, AsyncIterable[str], AsyncIterable[TextGenerationStreamOutput]]:
         """
         Given a prompt, generate the following text.
 
